src/Workspaces/VisualBasic/Portable/CaseCorrection/VisualBasicCaseCorrectionService.Rewriter.vb

```diff
@@ -150,6 +155,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CaseCorrection
                 Return CaseCorrectIdentifierIfNamesDiffer(token, newToken, symbol)
             End Function
 
+            Private Shared Function IsNamespaceOrTypeRelatedSymbol(symbol As ISymbol) As Boolean
+                Return TypeOf symbol Is INamespaceOrTypeSymbol OrElse
+                    (TypeOf symbol Is IAliasSymbol AndAlso TypeOf DirectCast(symbol, IAliasSymbol).Target Is INamespaceOrTypeSymbol) OrElse
+                    (symbol.IsKind(SymbolKind.Method) AndAlso DirectCast(symbol, IMethodSymbol).MethodKind = MethodKind.Constructor)
```

Comment:
That and for Attributes.

Eg: Attributes,
```VB
Class SampleAttribute
    inherits Attribute
End Class

<sample>
Class D
End Class
```

```VB
Class SampleAttribute
    inherits Attribute
End Class

<Sample>
Class D
End Class
```

We have tests for these cases as well in the code.